shoop/admin/templates/shoop/admin/products/_edit_media_form.jinja

```diff
@@ -0,0 +1,96 @@
+{% from "shoop/admin/macros/general.jinja" import content_block %}
+{% from "shoop/admin/macros/multilanguage.jinja" import language_dependent_content_tabs %}
+{% set media_form = form[form_def.name] %}
+
+{% macro render_media_form(f, media_form, idx) %}
+    {% for h in f.hidden_fields() %}
+        {{ h|safe }}
+    {% endfor %}
+    {%- set is_image_form = media_form.prefix == "images" %}
```

Comment:
You assign this value from `media_form.prefix == "images"`, then only use it in one place, where the `media_form.prefix == "images"` expression is used like 5 times.